src/rmats2sashimiplot/rmats2sashimiplot.py

```diff
@@ -401,6 +424,12 @@ def main():
                 up_e = items[8]
                 dn_s = str(int(items[9])+1)
                 dn_e = items[10]
+                #### Get the minimum and maximum position to represent 

```

Comment:
Thanks for your contribution! We are reviewing and doing some tests of
these modify, once it's done we will merge this branch.

Thank you,

*Ting*

On Mon, Dec 28, 2015 at 6:53 PM, Chen Tong <notifications@github.com> wrote:

> In src/rmats2sashimiplot/rmats2sashimiplot.py
> <https://github.com/Xinglab/rmats2sashimiplot/pull/2#discussion_r48519324>
> :
>
> > @@ -401,6 +424,12 @@ def main():
> >                  up_e = items[8]
> >                  dn_s = str(int(items[9])+1)
> >                  dn_e = items[10]
> > +                #### Get the minimum and maximum position to represent
>
> This deals with errors listed below when genes located in negative strand.
>
>   File "/MPATHB/soft/anacond/bin/sashimi_plot", line 9, in <module>
>     load_entry_point('misopy==0.5.3', 'console_scripts', 'sashimi_plot')()
>   File "/MPATHB/soft/anacond/lib/python2.7/site-packages/misopy-0.5.3-py2.7-linux-x86_64.egg/misopy/sashimi_plot/sashimi_plot.py", line 276, in main
>     plot_label=plot_label)
>   File "/MPATHB/soft/anacond/lib/python2.7/site-packages/misopy-0.5.3-py2.7-linux-x86_64.egg/misopy/sashimi_plot/sashimi_plot.py", line 153, in plot_event
>     plot_label=plot_label)
>   File "/MPATHB/soft/anacond/lib/python2.7/site-packages/misopy-0.5.3-py2.7-linux-x86_64.egg/misopy/sashimi_plot/plot_utils/plot_gene.py", line 698, in plot_density_from_file
>     plot_title=plot_title)
>   File "/MPATHB/soft/anacond/lib/python2.7/site-packages/misopy-0.5.3-py2.7-linux-x86_64.egg/misopy/sashimi_plot/plot_utils/plot_gene.py", line 362, in plot_density
>     plot_mRNAs(tx_start, mRNAs, strand, graphcoords, reverse_minus)
>   File "/MPATHB/soft/anacond/lib/python2.7/site-packages/misopy-0.5.3-py2.7-linux-x86_64.egg/misopy/sashimi_plot/plot_utils/plot_gene.py", line 504, in plot_mRNAs
>     x = [graphcoords[s], graphcoords[e], graphcoords[e], graphcoords[s]]
> IndexError: index 576 is out of bounds for axis 0 with size 99
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Xinglab/rmats2sashimiplot/pull/2/files#r48519324>.
>